store/src/sqlite/model.rs

```diff
@@ -0,0 +1,1365 @@
+use std::{collections::BTreeSet, marker::PhantomData};
+
+use anyhow::{anyhow, Result};
+use async_trait::async_trait;
+use bytes::Bytes;
+use ceramic_core::{EventId, RangeOpen};
+use cid::Cid;
+use iroh_bitswap::Block;
+use iroh_car::{CarHeader, CarReader, CarWriter};
+use itertools::{process_results, Itertools};
+use multihash_codetable::{Code, MultihashDigest};
+use recon::{AssociativeHash, HashCount, InsertResult, Key, ReconItem, Sha256a};
+use sqlx::Row;
+use tracing::instrument;
+
+use crate::{DbTx, SqlitePool};
+
+/// Unified implementation of [`recon::Store`] and [`iroh_bitswap::Store`] that can expose the
+/// individual blocks from the CAR files directly.
+#[derive(Clone, Debug)]
+pub struct ModelStore<H>
```

Comment:
This file should be safe to delete.. I moved it to `sql/event.rs` and it looks like you applied to changes there.